### ISSUE TITLE
Convert namespaces from `UpperCamelCase` to `snake_case`

### ISSUE THREAD

**Issue by reyk** [butte-rs/butte#50]
Hi,

the official [Style guide](https://google.github.io/flatbuffers/flatbuffers_guide_writing_schema.html) says that namespaces are `UpperCamelCase`. This doesn't work so well with Rust module names that are in `snake_case` so flatc converts the namespace names and butte doesn't.

```
namespace HelloWorld;
```

`butte`:
```Rust
pub mod HelloWorld {
```

`flatc`:
```Rust
pub mod hello_world {
```

**Comment by cpcloud** [comment]
Closed by #56.

**Comment by cpcloud** [comment]
Thanks for the fix @reyk!
